NEWS.md

```diff
@@ -1,3 +1,7 @@
+# covidregionaldata 0.9.3
+
+This release completes all recommendations for the submission to JOSS.
```

Comment:
Ah so I was expecting us to jump from 0.9.1 -> 0.9.3 and so extend the news below. 

Really it needs a summary of what is in the release like the previous news summaries.